code/_onclick/hud/rendering/render_plate.dm

```diff
@@ -396,7 +396,6 @@
  * Other vars such as alpha will automatically be applied with the render source
  */
 /atom/movable/screen/plane_master/proc/generate_render_relays()
-	var/relay_loc = "CENTER"
 	// If we're using a submap (say for a popup window) make sure we draw onto it
```

Comment:
The ?. is needed because of testing cases, home can be null.
We can just rely on the plane master group's opinion here, ya get me?
```suggestion
	var/relay_loc = home?.relay_loc || "CENTER
	// If we're using a submap (say for a popup window) make sure we draw onto it
```